File: cardapio.cpp
#include<iostream>
using namespace std;
#include<string>
#include<conio.h>
#include<unistd.h>

int main(){
	
	char menu, esco, resp;
	int valcont = 0, diverCom = 5, diverBeb = 3;
	float temp = 1.5;
	bool flag0, flag1;
	
	string prato[diverCom] = {"Peixe..........r$ 18","Bife...........r$ 18","Frango.........r$ 12","Vegano.........r$ 10","Basico.........r$ 12"};
	string bebida[diverBeb] = {"Agua...........r$ 3","Cerveja........r$ 10","Refrigerante...r$ 5"};
	
	do{
		flag0 = 0;
				
		system("cls");
		cout << "####################" << endl;
		cout << "Opcoes:\n 1- Prato\n 2- Bebida" << endl;
		cout << "####################\n";
		menu = getch();
	
		switch(menu){
		
			case '1':
				do{
					flag1 = 0;
					system("cls");
					cout << "--------------------" << endl;
					for(int i = 0; i < diverCom; i++)
					cout << " " << i+1 << "-" << prato[i] << endl;
					cout << "--------------------" << endl;
				
					esco = getch();
					
						switch(esco){
							case '1':
								valcont += 18;
								break;
								
							case '2':
								valcont += 18;
								break;
								
							case '3':
								valcont += 12;
								break;
								
							case '4':
								valcont += 10;	
								break;
								
							case '5':
								valcont += 12;
								break;
							
							default:
								cout << "Responda uma das opcoes\n";
								sleep(temp);
								flag1 = 1;
						}
				}while(flag1);
						
				cout << "Deseja escolher mais coisas? s/n" << endl;
				resp = getch();
				if(resp == 's' || resp == 'S') flag0 = true;	
				break;
				
			case '2':
				do{
					flag1 = 0;
					system("cls");
					cout << "--------------------" << endl;
					for(int i = 0; i < diverBeb; i++)
					cout << " " <<  i+1 << "-" << bebida[i] << endl;
					cout << "--------------------" << endl;
					
					esco = getch();
						
						switch(esco){
							case '1':
								valcont += 3;
								break;
								
							case '2':
								valcont += 10;
								break;
								
							case '3':
								valcont += 5;
								break;
								
							default:
								cout << "Responda uma das opcoes\n";
								sleep(temp);
								flag1 = 1;
						}
				}while(flag1);
				
				cout << "Deseja escolher mais coisas? s/n" << endl;
				resp = getch();
				if(resp == 's' || resp == 'S') flag0 = true;
				break;
				
			default:
				cout << "Responda uma das opcoes ofertadas\n";
				sleep(temp);
				flag0 = 1;
		}
	} while(flag0);
	
	system("cls");
	cout << "##############################" << endl;
	cout << "TOTAL DA CONTA : r$ " << valcont << endl << "OBRIGADO POR COMPRAR CONOSCO!" << endl;
	cout << "##############################";
}

File: cardapio.py
menu = ('Prato','Bebida')
prato = ('Pizza','Hamburguer','Pastel')
priceP = (30,8,5)
bebida = ('Agua','Refri','Cerveja')
priceB = (2,5,10)
valor = 0
flag = 1
while flag:
    flag = 0
    print('{:#^30}'.format(' MENU '))
    for c in range(0,2):
        print('{}-{}'.format(c + 1, menu[c]))
    print('{:#^30}'.format(' MENU '))
    escom = int(input(''))

    if escom == 1:
        print('{:#^30}'.format(' PRATO '))
        for c in range(0,3):
            print('{}-{:.<20}R$ {}'.format(c + 1,prato[c],priceP[c]))
        print('{:#^30}'.format(' PRATO '))
        esco = int(input(''))
        if esco == 1: 
            valor += priceP[esco-1]
        if esco == 2: 
            valor += priceP[esco-1]
        if esco == 3: 
            valor += priceP[esco-1]
        
    if escom == 2:
        print('{:#^30}'.format(' BEBIDA '))
        for c in range(0,3):
            print('{}-{:.<20}R$ {}'.format(c + 1,bebida[c],priceB[c]))
        print('{:#^30}'.format(' BEBIDA '))
        esco = int(input(''))
        if esco == 1: 
            valor += priceB[esco-1]
        if esco == 2: 
            valor += priceB[esco-1]
        if esco == 3: 
            valor += priceB[esco-1]
    rep = input('Deseja escolher algo mais? s/n ')
    if rep == 's':
        flag = 1
    
print('{:#^30}'.format(' CONTA '))
print('O valor da conta ficou em R${}'.format(valor))
print('{:#^30}'.format(' CONTA '))
